core/src/main/scala/org/apache/spark/sql/TiExtensions.scala

```diff
@@ -65,8 +65,8 @@ object TiExtensions {
       .find(pair => TiCatalog.className.equals(pair._2)) match {
       case None =>
         logger.error(
-          "TiSpark is not work in catalog plugin mode! Please add catalog config in spark conf")
-        throw new TiInternalException("TiSpark is not work in catalog plugin mode")
+          "TiSpark must work in catalog plugin mode! Please add catalog config in spark conf")
+        throw new TiInternalException("TiSpark must work in catalog plugin mode")
```

Comment:
The same as log, since the exception message is the first message that the user notices.

Reply:
i forget change log